### PR TITLE
DROOLS-2555 : Broken welcome page layout in IE11

### PR DESCRIPTION
**Pull request by Rikkola**
https://issues.jboss.org/browse/DROOLS-2555

Bundle, but can be merged individually.
https://github.com/kiegroup/kie-wb-distributions/pull/761
https://github.com/kiegroup/kie-wb-common/pull/1797

### CODE REVIEW

File: kie-wb-common-screens/kie-wb-common-home/kie-wb-common-home-client/src/main/java/org/kie/workbench/common/screens/home/client/widgets/home/HomeView.less [hunk 1]
```diff
@@ -74,6 +74,7 @@
   }
 
   .kie-hero-card {
+    height : 210px;
```
Comment (jomarko) [inline kie-wb-common-screens/kie-wb-common-home/kie-wb-common-home-client/src/main/java/org/kie/workbench/common/screens/home/client/widgets/home/HomeView.less]: Just formatting:
height: 210px;

File: kie-wb-common-screens/kie-wb-common-home/kie-wb-common-home-client/src/main/java/org/kie/workbench/common/screens/home/client/widgets/home/HomeView.less [hunk 2]
```diff
@@ -114,6 +115,7 @@
   .kie-hero-card__text {
     flex: 3 0;
     text-align: left;
+    width:230px;
```
Comment (jomarko) [inline kie-wb-common-screens/kie-wb-common-home/kie-wb-common-home-client/src/main/java/org/kie/workbench/common/screens/home/client/widgets/home/HomeView.less]: Just formatting:
width: 230px;

**Comment by Rikkola** [comment]
@jomarko Updated the PR.

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Comment by Rikkola** [comment]
jenkins please retest this

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Pull request merged**